packages/@pollyjs/adapter-node-http/src/utils/http-wrapper.js

```diff
@@ -0,0 +1,35 @@
+import http from 'http';
+import https from 'https';
+
+import TransportWrapper from './transport-wrapper';
+
+export default class HttpWrapper {
+  constructor({ adapter, options }) {
+    this.transports = [];
+
+    adapter.assert(
+      'There should be at least one transport enabled',
```

Comment:
`Invalid transports provided. At least one supported transport must be specified.`

Reply:
This can also be moved to the adapter's onConnect. That way you can assume that the HttpWrapper always receives valid options. Totally up to you though 😄

Reply:
Done in 25589639be340c56e0fcd56d85d13c36585fde2f